### DISCUSSION TITLE
What should I do with the Twitter account?

### DISCUSSION THREAD

**Discussion by marienfressinaud** [FreshRSS/FreshRSS#5125] [Strategy]
Hi,

I’ve opened the Twitter account ([@FreshRSS](https://twitter.com/FreshRSS)), not long before the release of FRSS 1.0. I’ve never been very active with it, and I mostly retweet or like tweets that mention FreshRSS.

I've been considering to delete or transfer the account for a long time, but never took the time. Last year I've deleted my personal account, and the recent events (aka Musk's takeover of Twitter) pushed me to close my professional account. The FreshRSS account is the last one I still manage, but I’d like to stop.

There are two options:

1. someone (preferably someone already involved in the community) wants to animate the account: I can transfer the credentials
2. nobody wants to do it, and I would delete it

I’ll keep this discussion opened for a maximum of a month.

/cc @Alkarex @Frenzie @aledeg @math-GH

**Comment by math-GH** [comment]
In general I am/was right active on social media like Twitter. Since Musk have been take over Twitter I moved to Mastodon and use Twitter more or less inactive. (I do not want to "support" Twitter anymore with content, comments and reading)

I would recommend to keep the Twitter account, write a message that the account is not anymore actively used. Please do not delete the account because scammers would register a new account with free account name and make bad things.  (see: https://lifehacker.com/dont-delete-your-twitter-account-do-this-instead-1849779710)

In general I am interested in to have an official FreshRSS account on Mastodon. If wished I could administrate the account (I am also open to take over the Twitter account to keep the account inactive)

**Reply by marienfressinaud** [reply]
You're right, we should not delete the account. I'll write a message to tell people that it is inactive and redirect them to GitHub if they need support :+1: 

Let me know if you want to take over the account, I'll transfer the credentials to you (if the others agree ;))

**Reply by Frenzie** [reply]
Works for me. :-)

**Reply by Alkarex** [reply]
I agree

**Reply by aledeg** [reply]
I am ok with that too.

**Comment by Frenzie** [comment]
I wouldn't really keep it active but I'm probably fine safeguarding it while it autoposts release info or something.

**Comment by aledeg** [comment]
I've never used Twitter, and I am not active on social medias. I have no idea if this account has some value. I've just learn its existence.
I do not want to manage it and I understand why you want to end it.

**Comment by marienfressinaud** [comment]
I'm closing this discussion since we agreed to make the account inactive: https://twitter.com/FreshRSS/status/1628453287518167042

@math-GH I'm sending you an email with the credentials (in a temporary PrivateBin)
